Documentation/How-to-start-your-app-automatically-on-boot-using-systemd.md

```diff
@@ -0,0 +1,151 @@
+# How to start app automatically on boot using systemd
+
+## Create your app
+
+For the purpose of this document let's assume you have deployed your app under:
+
+`
+/home/pi/myiotappfolder/myiotapp
+`
+
+The app name is `<myiotapp>` in this documentation you would replace the entire string including the carrots in your files.
+
+Make sure to make your app ```<myiotapp>``` is executable by using the command:
+
+```shell
+# Requires root permissions
```

Comment:
if this app runs as root you should mention that app and all its dependencies should be made writable for root only

Reply:
you already did in the bottom, make sure to add a link just before this

Reply:
Added a link